modules/indexing/src/test/java/org/apache/ignite/testsuites/IgniteBinaryCacheQueryTestSuite2.java

```diff
@@ -82,6 +84,9 @@
  */
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
+    SqlMergeTest.class,
+    SqlMergeOnClientNodeTest.class,
```

Comment:
Were these test classes added to two test suites intentionally? (here and IgniteBinaryCacheQueryTestSuite)